ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/AddMetadata.java

```diff
@@ -70,11 +31,13 @@
       byte[] metadata;
       try {
         // Get attributes as bytes, throws IOException
-        metadata = Json.asBytes(attributesToMetadataPayload(msg.getAttributeMap()));
+        metadata = Json.asBytes(com.mozilla.telemetry.ingestion.core.transform.AddMetadata
```

Comment:
We may want to choose a different name for this class in `core`. Perhaps `MetadataPayload` or `NestedMetadata`. That would avoid having this fully-qualified class reference and avoid potential confusion between the two classes in the future.

Reply:
`NestedMetadata` sounds good to me